config/src/test/java/org/springframework/security/config/annotation/web/configurers/RememberMeConfigurerTests.java

```diff
@@ -453,4 +456,35 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
 			// @formatter:on
 		}
 	}
+
+	@Test
+	public void getWhenRememberMeCookieThenAuthenticationIsRememberMeAuthenticationTokenWithFallbackKeyConfiguration()
+			throws Exception {
+		this.spring.register(FallbackRememberMeKeyConfig.class).autowire();
+
+		MvcResult mvcResult = this.mvc.perform(post("/login")
+				.with(csrf())
+				.param("username", "user")
+				.param("password", "password")
+				.param("remember-me", "true"))
+				.andReturn();
+		Cookie rememberMeCookie = mvcResult.getResponse().getCookie("remember-me");
+
+		this.mvc.perform(get("/abc")
+				.cookie(rememberMeCookie))
+				.andExpect(authenticated().withAuthentication(auth ->
+						assertThat(auth).isInstanceOf(RememberMeAuthenticationToken.class)));
+	}
+
+	@EnableWebSecurity
+	static class FallbackRememberMeKeyConfig extends RememberMeConfig {
+
+		@Override
+		protected void configure(HttpSecurity http) throws Exception {
+			// @formatter:off
+			http.rememberMe()
```

Comment:
I believe I misled you in my previous review, we will need additional configuration here.
```
// @formatter:off
http
	.authorizeRequests()
		.anyRequest().hasRole("USER")
		.and()
	.formLogin()
		.and()
	.rememberMe()
		.rememberMeServices(new TokenBasedRememberMeServices("key", userDetailsService()));
// @formatter:on
```